src/clj/fluree/db/query/sparql/translator.cljc

```diff
@@ -446,7 +444,10 @@
 (defmethod parse-term :Filter
   ;; Filter ::= <'FILTER'> WS Constraint
   [[_ constraint]]
-  [:filter [(parse-term constraint)]])
+  (let [parsed-constraint (parse-term constraint)]
+    (if (contains? #{"exists" "not-exists"} (first parsed-constraint))
```

Comment:
Because FQL implements `exists` and `not-exists` as top-level where patterns instead of as subordinate to a `filter` pattern like SPARQL I had to do a little manual unpacking of the parse tree in the translator here.